pro-dental-care-api/src/main/java/org/cc/prodentalcareapi/model/PatientBilling.java

```diff
@@ -0,0 +1,71 @@
+package org.cc.prodentalcareapi.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.persistence.Temporal;
+import jakarta.persistence.TemporalType;
+import java.util.Date;
+
+
+@Entity
+@Table(name = "payment_option")
```

Comment:
It looks like you swapped the names of these tables? Also the name of the billing table is just "bills". We don't need to rename the class but we could just specify it in the name property.